NBitcoin/RPC/RPCClient.Wallet.cs

```diff
@@ -91,40 +93,281 @@ wallet			 getbalance
 		wallet			 getnewaddress
 		wallet			 getrawchangeaddress
 		wallet			 getreceivedbyaccount
-		wallet			 getreceivedbyaddress		 Yes
+		wallet			 getreceivedbyaddress		Yes
 		wallet			 gettransaction
 		wallet			 getunconfirmedbalance
 		wallet			 getwalletinfo
 		wallet			 importprivkey				Yes
 		wallet			 importwallet
 		wallet			 importaddress				Yes
 		wallet			 keypoolrefill
-		wallet			 listaccounts				 Yes
-		wallet			 listaddressgroupings		 Yes
+		wallet			 listaccounts				Yes
+		wallet			 listaddressgroupings		Yes
 		wallet			 listlockunspent
 		wallet			 listreceivedbyaccount
 		wallet			 listreceivedbyaddress
 		wallet			 listsinceblock
 		wallet			 listtransactions
-		wallet			 listunspent				  Yes
-		wallet			 lockunspent				  Yes
+		wallet			 listunspent				Yes
+		wallet			 loadwallet					Yes
+		wallet			 lockunspent				Yes
 		wallet			 move
 		wallet			 sendfrom
 		wallet			 sendmany
 		wallet			 sendtoaddress
 		wallet			 setaccount
 		wallet			 settxfee
 		wallet			 signmessage
+		wallet			 unloadwallet				Yes
 		wallet			 walletlock
 		wallet			 walletpassphrasechange
-		wallet			 walletpassphrase			yes
+		wallet			 walletpassphrase			Yes
 		wallet			 walletprocesspsbt
 		wallet			 walletcreatefundedpsbt
 	*/
-	public partial class RPCClient
+
+	public interface IRPCWalletClient
```

Comment:
I really don't like this approach for interfaces.

Interfaces with many methods are a code smell, and everytime we add one method it will be a breaking change, I would prefer you remove the interface.